src/components/Checkbox/Checkbox.js

```diff
@@ -39,15 +39,15 @@ export const Checkbox = forwardRef(
     const mergedInputRef = useMergeRefs({ refs: [ref, inputRef] });
     const overrideClassName = backwardCompatibilityForProperties([className, componentClassName]);
     const onMouseUpCallback = useCallback(() => {
-      const input = mergedInputRef.current;
+      const input = inputRef.current;
```

Comment:
The way I used mergedInputRef was wrong, now the mergedInputRef is passed to the input as ref but everywhere I am keep using in the ref which I know that it have a value.
Using mergedInputRef as ref cause bug on the checkbox component

Reply:
I will add documentation about this mistake in storybook/ the hook code on a separate PR